virtualizarr/readers/hdf.py

```diff
@@ -0,0 +1,206 @@
+from typing import List, Mapping, Optional
+
+import fsspec
```

Comment:
Does one need fsspec if reading a local file? Is there any other way to read from S3 without fsspec at all?

Reply:
Not with a filesystem-like API. You would have to use boto3 or aiobotocore directly.

This is one of the great virtues of fsspec and is not to be under-valued.